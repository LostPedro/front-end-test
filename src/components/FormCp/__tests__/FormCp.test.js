import React from 'react'
import {render, cleanup} from '@testing-library/react'
import {FormCp} from '../'

afterEach(cleanup)

it('should take a snapshot', () => {
  const {asFragment} = render(<FormCp />)

  expect(asFragment(<FormCp />)).toMatchSnapshot()
})
