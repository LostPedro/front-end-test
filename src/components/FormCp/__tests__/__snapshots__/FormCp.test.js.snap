// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="form-component"
  >
    <form
      class="form-component-form"
    >
      <div
        class="form-component-all-inputs-container"
      >
        <div
          class="form-component-input-container"
        >
          <div
            class="input-component"
          >
            <label
              class=""
              for="name"
            >
              Nome da pessoa compradora
            </label>
            <input
              class="input-component-input"
              id="name"
              name="name"
              required=""
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          class="form-component-input-container"
        >
          <div
            class="input-component"
          >
            <label
              class=""
              for="cpf"
            >
              CPF
            </label>
            <input
              class="input-component-input"
              id="cpf"
              name="cpf"
              required=""
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          class="form-component-input-container"
        >
          <div
            class="input-component"
          >
            <label
              class=""
              for="cardNumber"
            >
              N° do cartão
            </label>
            <input
              class="input-component-input"
              id="cardNumber"
              name="cardNumber"
              required=""
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          class="form-component-input-container"
        >
          <div
            class="input-component"
            style="width: 58.53%;"
          >
            <label
              class=""
              for="expireDate"
            >
              Data de expiração
            </label>
            <input
              class="input-component-input"
              id="expireDate"
              name="expireDate"
              required=""
              type="text"
              value=""
            />
          </div>
          <div
            class="input-component"
            style="width: 39%;"
          >
            <label
              class=""
              for="cvv"
            >
              CVV
            </label>
            <input
              class="input-component-input"
              id="cvv"
              name="cvv"
              required=""
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          class="form-component-input-container"
        >
          <div
            class="input-component"
          >
            <label
              class=""
              for="transactionAmount"
            >
              Valor da transação
            </label>
            <input
              class="input-component-input"
              id="transactionAmount"
              name="transactionAmount"
              required=""
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
      <div
        class="form-component-register-button-wrapper"
      >
        <button
          class="button-component-disabled"
          disabled=""
          type="submit"
        >
          <div />
          Criar transação
        </button>
      </div>
    </form>
  </div>
</DocumentFragment>
`;
